<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Digraphform</class>
 <widget class="QDialog" name="Digraphform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(5,2,13);</string>
  </property>
  <widget class="QPushButton" name="InsVertice">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: #33FF99;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputVertice">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
border:2px;
border-style:solid;
border-color:rgb(28,19,42);
border-radius: 8px;
}

QTextEdit:focus {
    background-color: #DDD;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px;
color: #DDDDDD;</string>
   </property>
   <property name="text">
    <string>Insertar Vertice:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px;
color: #111;
background: #9DAABC;</string>
   </property>
   <property name="text">
    <string>Insertar Arista</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px;
color: #DDDDDD;</string>
   </property>
   <property name="text">
    <string>Vertice origen -&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>380</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px;
color: #DDDDDD;</string>
   </property>
   <property name="text">
    <string>Vertice Destino -&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px;
color: #DDDDDD;</string>
   </property>
   <property name="text">
    <string>Peso de la arista -&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputVO">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>350</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
border:2px;
border-style:solid;
border-color:rgb(28,19,42);
border-radius: 8px;
}

QTextEdit:focus {
    background-color: #DDD;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputVD">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>390</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
border:2px;
border-style:solid;
border-color:rgb(28,19,42);
border-radius: 8px;
}

QTextEdit:focus {
    background-color: #DDD;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputPeso">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>430</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
border:2px;
border-style:solid;
border-color:rgb(28,19,42);
border-radius: 8px;
}

QTextEdit:focus {
    background-color: #DDD;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InsArista">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>390</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: #33FF99;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Insertar Arista</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDijkstra">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>620</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: #33FF99;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Dijkstra</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFloyd">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>560</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: #33FF99;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Floyd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWarshall">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>560</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: #33FF99;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Warshall</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>681</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>649</width>
      <height>679</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelGrafico">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_Draw">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>480</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(28,19,42);
    border-style: outset;
    border-width: 0px;
    border-radius: 8px;
    border-color: beige;
    font: bold 18px;
	color: #33FF99;
    padding: 6px;
}</string>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputDij">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>560</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit{
border:2px;
border-style:solid;
border-color:rgb(28,19,42);
border-radius: 8px;
}

QTextEdit:focus {
    background-color: #DDD;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>570</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px;
color: #DDDDDD;</string>
   </property>
   <property name="text">
    <string>Elegir Nodo:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(5,2,13);</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 25pt &quot;Cooper Black&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string>GRAFOS DIRIGIDOS</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #9DAABC;</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14px;
/*color: #DDDDDD;*/
</string>
    </property>
    <property name="text">
     <string>Insertar Vertice:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
